### PR TITLE
Documentation of the function init_coregmrpopup

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Documentation of the function init_coregmrpopup. Description and inputs in need of completion.

### CODE REVIEW

File: ea_init_coregmrpopup.m [hunk 1]
```diff
@@ -1,4 +1,19 @@
 function ea_init_coregmrpopup(handles,refine)
+%
```
Comment (Daniel-Tojal) [inline ea_init_coregmrpopup.m]: Please provide a description of the function

File: ea_init_coregmrpopup.m [hunk 2]
```diff
@@ -1,4 +1,19 @@
 function ea_init_coregmrpopup(handles,refine)
+% 
+%
+% USAGE:
+%
+%    ea_init_coregmrpopup(handles,refine)
+%
+% INPUTS:
```
Comment (Daniel-Tojal) [inline ea_init_coregmrpopup.m]: Please provide a description of the inputs

File: ea_init_coregmrpopup.m (hunk 1 above)
Comment (ningfei) [reply ea_init_coregmrpopup.m]:
```
% Initialize coregistration methods popupmenu.
%
% INPUTS:
%    handles: LEAD GUI handle
%    refine: deprecated.
```

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal and @ningfei 👍

**Pull request merged**